CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,84 @@
+# Code of Conduct
+
+## 1. Purpose
+
+A primary goal of Design Systems Coalition NYC is to be inclusive to the largest number of members, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status, and religion (or lack thereof).
```

Comment:
- Opened a pr to add reference about building requirements and link to this CoC in the hosting guidelines: https://github.com/design-systems-coalition-NYC/meetup/pull/32
- Added "check building accessibility" to event checklist template: https://github.com/design-systems-coalition-NYC/meetup/blob/master/event-checklist.md